Src/WitsmlExplorer.Frontend/components/Modals/ModalDialog.tsx

```diff
@@ -70,67 +69,75 @@ const ModalDialog = (props: ModalDialogProps): React.ReactElement => {
         variant="contained"
       >
         {confirmText ?? "Save"}
-      </Button>
+      </StyledButton>
     ) : (
       <></>
     ),
     showCancelButton ? (
-      <Button key={"cancel"} disabled={isLoading} onClick={onCancel} color={confirmColor ?? "primary"} variant="outlined">
+      <StyledButton key={"cancel"} disabled={isLoading} onClick={onCancel} color={confirmColor ?? "primary"} variant="outlined">
         Cancel
-      </Button>
+      </StyledButton>
     ) : (
       <></>
     ),
-    <Button key={"delete"} disabled={isLoading} onClick={onDelete} color={"secondary"} variant="outlined" align={"right"}>
+    <StyledButton
+      key={"delete"}
+      disabled={isLoading}
+      onClick={onDelete}
+      color={"danger"}
+      variant="contained"
+      align={"right"}
+      style={{ marginLeft: "auto", margin: "0.5em", float: "right" }}
+    >
       Delete
-    </Button>
+    </StyledButton>
   ];
 
   return (
-    <Dialog onKeyDown={onKeyPress} open={displayModal} fullWidth maxWidth={width}>
-      <Title>{heading}</Title>
+    <Dialog onKeyDown={onKeyPress} open={displayModal} style={{ width: width }}>
+      <Dialog.Header>
+        <Dialog.Title>{heading}</Dialog.Title>
+      </Dialog.Header>
+
       <Content>
         {content}
         {errorMessage && <ErrorMessage>{errorMessage}</ErrorMessage>}
-        <DialogActions>
-          {buttons[switchButtonPlaces ? 1 : 0]}
-          {buttons[switchButtonPlaces ? 0 : 1]}
-          {isLoading && <CircularProgress size="1.5rem" />}
-          {onDelete && buttons[2]}
-        </DialogActions>
       </Content>
+      <Dialog.Actions style={{ width: "100%" }}>
+        {isLoading ? (
+          <StyledButton>
```

Comment:
I think the button with progress dots should have style={ cursor:"not-allowed" } to indicate that it is not clickable if we are not going to use set it to disabled to preserve the style

Reply:
I don't think MUI / EDS supports changing of cursor

Reply:
Fixed